[PATCH] fix build without curses

bmon supports a --disable-curses configure option, but still #error
out if curses wasn't detected, even though the rest of the code is
written to work without curses support.

Fix it by removing the bogus #error line.

Signed-off-by: Peter Korsgaard <jacmet@sunsite.dk>
---
 include/bmon/config.h |    2 --
 1 file changed, 2 deletions(-)

Index: bmon-2.1.0/include/bmon/config.h
===================================================================
--- bmon-2.1.0.orig/include/bmon/config.h
+++ bmon-2.1.0/include/bmon/config.h
@@ -118,8 +118,6 @@
 #  endif /* !HAVE_CURSES */
 #elif defined HAVE_CURSES
 #  include <curses.h>
-#else
-#  error "*** ERROR: Neither ncurses nor curses is present on your system. ***"
 #endif /* HAVE_[N]CURSES */
 
 #if defined HAVE_RRD
